XCUITests/FxScreenGraph.swift

```diff
@@ -478,6 +479,11 @@ func createScreenGraph(for test: XCTestCase, with app: XCUIApplication) -> MMScr
             }
         }
         
+        screenState.gesture(forAction: Action.CloseURLBarOpen, transitionTo: HomePanelsScreen) {_ in
+            sleep(1)
+            app.buttons["urlBar-cancel"].tap()
+        }
+        
```

Comment:
I've added the `Action.CloseURLBarOpen` here because when I tried to add it to `map.addScreenState(URLBarOpen) ` it failed to close the URL Bar focus.

Reply:
Sounds good! it may be a better place than the other one.. but I will try to remove the sleep from there, it will make tests slower... I would suggest to add a `waitForExistence(app.buttons["urlBar-cancel"], timeout: 5) ` in the tests before calling the new action defined:
`navigator.performAction(Action.CloseURLBarOpen)`

that way you will wait for that element if it is needed but you won't if it is there already... Please let me know if that works